### PR TITLE
Add Darwin availability annotations for some new features.

### PR DESCRIPTION
**Pull request by bzbarsky-apple**

### CODE REVIEW

**Review by ksperling-apple — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
Fast-tracking platform-specific fix with platform owner review.

File: src/darwin/Framework/CHIP/templates/availability.yaml [hunk 1]
```diff
@@ -9694,6 +9694,25 @@
           - ThreadBorderRouterManagement
           - ThreadNetworkDirectory
           - WiFiNetworkManagement
+      attributes:
+          OccupancySensing:
+              # Targeting Fall 2024
```
Comment (woody-apple) [inline src/darwin/Framework/CHIP/templates/availability.yaml:9699]:
```suggestion
              # Targeting 1.4
```

File: src/darwin/Framework/CHIP/templates/availability.yaml [hunk 2]
```diff
@@ -9694,6 +9694,25 @@
...
+              - HoldTimeLimits
+      commands:
+          UnitTesting:
+              # Ideally none of UnitTesting would be exposed as public API, but
+              # for now just start doing that for new additions to it.
+              - StringEchoRequest
+              - StringEchoResponse
+      structs:
+          OccupancySensing:
+              # Targeting Fall 2024
```
Comment (woody-apple) [inline src/darwin/Framework/CHIP/templates/availability.yaml:9710]:
```suggestion
              # Targeting 1.4
```

File: src/darwin/Framework/CHIP/templates/availability.yaml [hunk 3]
```diff
@@ -9694,6 +9694,25 @@
...
+              - HoldTimeLimitsStruct
+      bitmaps:
+          OccupancySensing:
+              # Targeting Fall 2024
```
Comment (woody-apple) [inline src/darwin/Framework/CHIP/templates/availability.yaml:9714]:
```suggestion
              # Targeting Fall 1.4
```

**Review by andy31415 — APPROVED** [review]

**Pull request merged**